### DISCUSSION TITLE
Are there CUDA accelerated segmentaton evaluation metrics?

### DISCUSSION THREAD

**Discussion by jakubMitura14** [Project-MONAI/MONAI#3223] [General]
Hello I am  creating Julia language package for CUDA accelerated medical segmentation evaluations like DICE, JAccard mutual information Mahalinobis distance, Housedorff distance etc. I started creating this becouse I was unable to find any CUDA accelerated medical segmentation package for evaluation - but maybe I am wrong?
Anybody knows GPU accelerated medical segmentation evaluation package, or is it possible in MONAI?

**Comment by Nic-Ma** [comment]
Hi @jakubMitura14 ,

Thanks for your interest here.
MONAI metrics are based on PyTorch Tensor (model output, label, etc.) which can execute on GPU with CUDA.

Thanks.
